packages/tocco-ui/src/Table/TableHeader.js

```diff
@@ -51,11 +52,10 @@ const TableHeader = ({columns, data, onColumnPositionChange, onSortingChange, ta
             })}
           >
             <StyledDnD
```

Comment:
Now you have just removed the `isDraggedOver` property. But inside `SytledDnD` you still use it:

https://github.com/tocco/tocco-client/blob/c3f0d627918234f651c6e97d7b0001ece13f31f9/packages/tocco-ui/src/Table/StyledTable.js#L31